resources/Arduino/devkit/c_cpp_properties.json

```diff
@@ -0,0 +1,17 @@
+{
+    "configurations": [
+        {
+            "name": "Linux",
+            "includePath": [
+                "${workspaceFolder}",
+                "/root/.arduino15/packages/AZ3166/tools/**",
+                "/root/.arduino15/packages/AZ3166/hardware/stm32f4/**"
```

Comment:
package version missing

Reply:
Since there will be only one version of AZ3166 package in one particular project's container, I delete the `version` information and use recursive mark`**`.

Reply:
Adding `version` in the `includePath` will makes it better for user to include AZ3166 libraries. They can use `#include <AZ3166.h>` instead of use `#include <1.6.2/AZ3166.h>`. 
Currently use hard-coded version `1.6.2`. We can add this feature when we accomplish the library and image manager.

Reply:
consider to still keep the version

Reply:
Why?